src/Components/Publishing/VideoPlayer/__test__/FullscreenHelpers.test.js

```diff
@@ -0,0 +1,136 @@
+import {
+  addFSEventListener,
+  exitFullscreen,
+  fullscreenEnabled,
+  isFullscreen,
+  removeFSEventListener,
+  requestFullscreen
+} from "../FullscreenHelpers"
+
+describe("Fullscreen helper methods", () => {
+  let event
+
+  beforeEach(() => {
+    event = new Event("fullscreenchange")
+    Object.defineProperty(document, 'fullscreenElement', {
+      configurable: true,
+      enumerable: true,
+      value: true,
+      writable: true
+    })
+    Object.defineProperty(document, 'fullscreenEnabled', {
+      configurable: true,
+      enumerable: true,
+      value: true,
+      writable: true
+    })
```

Comment:
@kanaabe - curious why this was needed?

Reply:
Ooh yeah I believe this was an artifact of when this test was in typescript. We can definitely just use `document.fullscreenElement = true` now! Good catch ⚾️